agoradatatools/etl/load.py

```diff
@@ -85,7 +110,9 @@ def load(file_path: str, provenance: list, destination: str, syn=None):
         activity = Activity(used=provenance)
     except ValueError:
         print(str(provenance) + " has one or more invalid syn ids")
-        return
+        return (
+            None  # added to be more explicit and consistent with the rest of the script
```

Comment:
I changed `return` to `return None` in a few places in this function to be more explicit and to match the other functions